### PR TITLE
Multi cell mesh

### PR DESCRIPTION
**Pull request by vlukes**
This PR allows to use meshes containing the different cell types.

### CODE REVIEW

File: sfepy/examples/linear_elasticity/mixed_mesh.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
+r"""
+A linear elastic beam loaded with a continuous force. The FE mesh consists
+of hexehedral and tetrahedral elements.
+
+The displacement at the beam end, is compared to the reference
```
Comment (rc) [inline sfepy/examples/linear_elasticity/mixed_mesh.py:5]: Remove the comma?

**Comment by rc** [comment]
Thanks! LGTM, but maybe add a nice gallery view? :)

**Comment by vlukes** [comment]
> Thanks! LGTM, but maybe add a nice gallery view? :)

Yes, I would like to extend the example after I finish the implementation of wedge elements. So, please, wait :-)

**Comment by rc** [comment]
OK, But I merge this now?

**Comment by vlukes** [comment]
> OK, But I merge this now?

Yes, please, merge it.

**Pull request merged**